### ISSUE TITLE
How do you compute the mAP in Table 1 of the paper giving the unknown samples?

### ISSUE THREAD

**Issue by d12306** [Vastlab/Elephant-of-object-detection#5]
Hi, @akshay-raj-dhamija , thanks for your implementation, 

but how do you calculate the mAP of the detector giving the unknown samples in the test phase,  since TP, FP, TN, FN are determined based on the labels and the predicted boxes for each class in the test set.  But the class labels of the unknown samples do not overlap with those of the images in the training set, so these images and the objects in them are labeled the "background" in order to calculate the mAP?
